gflownet/gflownet.py

```diff
@@ -855,6 +862,7 @@ def test(self):
                 hist[tuple(x)] += 1
             z_pred = sum([hist[tuple(x)] for x in x_tt]) + 1e-9
             density_pred = np.array([hist[tuple(x)] / z_pred for x in x_tt])
+            corr = np.corrcoef(density_pred, density_true)
```

Comment:
This correlation is not the "old" correlation between logq and energy. That was not your intention was it?

Reply:
Yep it is not the old one. I just wanted an easy measure of the correlation between the frequency of states generated by the sampler (probability of sampling a state) vs the state's reward.